normalize_license_text/input_text_files/Adobe1.txt

```diff
@@ -0,0 +1,9 @@
+Adobe Systems Incorporated(r) Source Code License Agreement Copyright(c) 2006
```

Comment:
Can we move this to resources folder?

Reply:
These are again just example files to test the CLI of features. They aren't being used in the code. I am just using them to run `python normalize.py Adobe1.txt` . I will remove them if you dont find it useful for new users.

Reply:
Move them to either examples/ or resources/ folder